internal/services/automation/automation_powershell72_module_resource.go

```diff
@@ -320,6 +326,7 @@ func (r PowerShell72ModuleResource) Read() sdk.ResourceFunc {
 
 			output.Name = id.PowerShell72ModuleName
 			output.AutomationAccountID = module.NewAutomationAccountID(id.SubscriptionId, id.ResourceGroupName, id.AutomationAccountName).ID()
+			output.Tags = pointer.From(resp.Model.Tags)
```

Comment:
Like we do elsewhere for Tags in the read, can we nil check the Model and use the `tags.Flatten` function.
```suggestion
			if model := resp.Model; model != nil {
				output.Tags = tags.Flatten(model.Tags)
			}
			
```